client/components/LearnerViewDirective.js

```diff
@@ -71,6 +50,27 @@ tie.directive('learnerView', [function() {
               </div>
             </div>
           </div>
+          <div class="tie-question-ui">
+            <div class="tie-question-window">
+              <h3>Exercise {{exerciseNumber}}: {{title}}</h3>
```

Comment:
Per offline discussion, I see the value in keeping "Exercise {{exerciseNumber}}:" there and agree.